Tests/SwiftStdlibTests/StringExtensionsTests.swift

```diff
@@ -386,6 +386,18 @@ final class StringExtensionsTests: XCTestCase {
         XCTAssertEqual("Hello This Tests".count(of: "t", caseSensitive: false), 3)
     }
 
+    func testDominantLanguage() {
+        if #available(iOS 11.0, *, macOS 10.13, tvOS 11.0, watchOS 4.0) {
```

Comment:
This should be `@available(...)` at the function signature level, rather than internally in the implementation.

Reply:
@AdrianBinDC can you please fix this? ☝️